src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -968,15 +969,19 @@ internal class Context
 
             public IDictionary<string, string> Headers { get; set; }
 
+            public IDictionary<string, string> Parameters { get; set; }
+
             internal void RefineFinding(string ruleId)
             {
                 RuleId = ruleId;
                 ClearProperties();
             }
 
-            internal void RefineRequest(string uri, string method)
+            internal void RefineRequest(string protocol, string version, string target, string method)
```

Comment:
>RefineRequest [](start = 26, length = 13)

I'm not convinced of the need for these methods... might remove them later, but will talk to Scott first, who I think introduced this pattern.